gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerViewModel.kt

```diff
@@ -443,6 +428,16 @@ internal class AvatarPickerViewModel(
             .launchIn(viewModelScope)
     }
 
+    private fun avatarsObserver() {
```

Comment:
⛏️ I would rename it to `collectAvatars` or `observe` - the current name sounds like we would return or create the Observer.